docs-src/concepts/what-is-a-custom-data-converter.md

```diff
@@ -0,0 +1,30 @@
+---
+id: what-is-a-custom-data-converter
+title: What is a custom Data Converter?
+sidebar_label: Custom Data Converter
+description: A custom Data Converter is an implementation of the Data Converter with custom logic for payload conversion and payload encryption.
+
+tags:
+  - term
+  - explanation
+---
+
+A custom Data Converter is an implementation of the [Data Converter](/concepts/what-is-a-data-converter) with custom logic for payload conversion and payload encryption.
```

Comment:
This will make readers implement `DataConvderter` interface, which they should pretty much never do. We need to reformulate it to avoid this effect. I don't think we need even to give such a "definition" here of "What is a custom Data Converter?"
"User may customize data conversation logic by specifying own PayloadConverts and PayloadCodecs".
Or "A custom Data Converter is a customization of default Data Conversion logic with own PayloadConverts and PayloadCodecs" if we have to write definitions.

Reply:
Updated to:
`A custom Data Converter implements custom logic for payload conversion or payload encryption to customize the default [Data Converter](/concepts/what-is-a-data-converter).`